python/cudf/cudf/io/json.py

```diff
@@ -27,24 +28,28 @@ def read_json(
     if engine == "auto":
         engine = "cudf" if lines else "pandas"
 
-    is_single_filepath_or_buffer = ioutils.ensure_single_filepath_or_buffer(
-        path_or_data=path_or_buf, **kwargs,
-    )
-    if not is_single_filepath_or_buffer:
-        raise NotImplementedError(
-            "`read_json` does not yet support reading multiple files"
-        )
-
-    path_or_buf, compression = ioutils.get_filepath_or_buffer(
-        path_or_data=path_or_buf,
-        compression=compression,
-        iotypes=(BytesIO, StringIO),
-        **kwargs,
-    )
```

Comment:
I think this step is being used by `pd.read_json` aswell. Do we want to get rid of this here?

Reply:
I left it there. Just renamed the variables so they made more sense in the context of the changes. Just shows as removed because the variable names were changed.